framework/source/class/qx/bom/client/Css.js

```diff
@@ -426,18 +426,20 @@ qx.Bootstrap.define("qx.bom.client.Css",
     __isFilterSupported : function(filterClass, initParams)
     {
       var supported = false;
-      var value = "progid:" + filterClass + "(" + initParams + ");";
-      var el = document.createElement("div");
-      document.body.appendChild(el);
-      el.style.filter = value;
-
-      if (el.filters && el.filters.length > 0 &&
-        el.filters.item(filterClass).enabled == true)
+      if (qx.core.Environment.get("browser.name") != "firefox")
       {
-        supported = true;
-      }
-      document.body.removeChild(el);
+        var value = "progid:" + filterClass + "(" + initParams + ");";
+        var el = document.createElement("div");
+        document.body.appendChild(el);
+        el.style.filter = value;
 
+        if (el.filters && el.filters.length > 0 &&
+          el.filters.item(filterClass).enabled == true)
+        {
+          supported = true;
+        }
+        document.body.removeChild(el);
+      }
```

Comment:
I would suggest if we touch this, then to deprecate the functions and always return `false` 
see also https://developer.mozilla.org/en-US/docs/Web/CSS/-ms-filter

we are not talking about genuine filter here but a M$ hack not required since IE8!

doing the whole sing and dance just takes valuable time ... at execution time.

better still remove the entire function and add deprecation notices to the getFilterGradient and getFilterTextShadow  calls.